Fabulous.CodeGen/src/Fabulous.CodeGen/AssemblyReader/Reader.fs

```diff
@@ -111,10 +122,11 @@ module Reader =
         
         { FullName = tdef.FullName
           AssemblyName = tdef.Module.Assembly.Name.Name
-          CanBeInstantiated = not tdef.IsAbstract && ctor.IsSome && ctor.Value.Parameters.Count = 0
```

Comment:
That's just a hot take, but I think I added this check to avoid picking up secondary constructors that need some parameters.
Would be great to double check this with the new `ConstructorsParameters` instead of just removing the check.

Reply:
I added this check in a another place.